ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -165,6 +166,22 @@ private static void setTransformProperty(View view, ReadableArray transforms) {
     view.setRotationY((float) sMatrixDecompositionContext.rotationDegrees[1]);
     view.setScaleX((float) sMatrixDecompositionContext.scale[0]);
     view.setScaleY((float) sMatrixDecompositionContext.scale[1]);
+
+    double[] perspectiveArray = sMatrixDecompositionContext.perspective;
+
```

Comment:
When no perspective is set, on Android, you still see some perspective when doing "3d transformation" while on iOS it show as expected. Therefore you always get a different result on iOS and Android when setting perspective yourself. Do you see what I mean? I could provide some examples.

I would like to know what is the default perspective an android so I can adjust the transformation correctly for android.

Thank you so much for your great work, really appreciate it.